hr_grap_custom/models/hr_employee.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Coop IT Easy SCRL fs
+# Quentin DUPONT <quentin.dupont@grap.coop>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class Employee(models.Model):
+    _inherit = "hr.employee"
+
+    # Columns Section
+    mutual_insurance_state = fields.Selection(
+        [('affiliated', 'Affiliated'),
```

Comment:
Does this pass Black hooks? I thought it was forcing double quotes?

Reply:
I'm a bad student and didn't learn to use black hooks yet :astonished: 
I'm gonna ask @legalsylvain to show me :)